scripts/ExportScripts/export_profile_monolithic.py

```diff
@@ -43,13 +47,16 @@
 process_command(['cmake', '--build',  args.mono_build_path, '--target', f'{project_name}.GameLauncher', '--target', f'{project_name}.ServerLauncher', '--target', f'{project_name}.UnifiedLauncher', '--config', 'profile'])
 
 #bundle content (WIP)
-# process_command([asset_bundler_exe, f'--project-path="{str(o3de_context.project_path)}"'])
+# process_command([asset_bundler_exe, 'assetList','--assetListFile {}',f'--project-path="{str(o3de_context.project_path)}"'])
 
 #copy artifacts to output directory
 
 pc_pak_path = args.output_path / 'Cache' / 'pc'
 
 os.makedirs(pc_pak_path, exist_ok=True)
+os.makedirs(args.output_path, exist_ok=True)
 # os.makedirs(args.output_path / 'Gems' / 'AWSCore')
-process_command(['copy', str(o3de_context.project_path / 'AssetBundling' / 'Bundles' / '*.pak'), str(pc_pak_path)])
-process_command(['copy', str(args.mono_build_path / 'bin' / 'profile' / '*.*'), str(args.output_path)])
+# process_command(['copy', str(o3de_context.project_path / 'AssetBundling' / 'Bundles' / '*.pak'), str(pc_pak_path)])
+source_path = str(args.mono_build_path / 'bin' / 'profile')
+output_path = str(args.output_path)
+process_command(f'xcopy {source_path} {output_path}', cwd=o3de_context.project_path)
```

Comment:
There is no reason to make this code less portable.

CMake CLI has a [copy](https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-E-arg-copy) command that works across all platforms.
It can be invoked via the `cmake -E copy_if_different` or `cmake -E copy_directory_if_different` command